frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureReleasePlanCard/FeatureReleasePlanCard.tsx

```diff
@@ -47,37 +68,88 @@ const StyledTopRow = styled('div')(({ theme }) => ({
     width: '100%',
 }));
 
+const UseButton = styled(Button)(({ theme }) => ({
```

Comment:
Wouldn't buttons be easier to just use as variants of the MUI button? E.g. `<Button variant="outlined">` and `variant="contained"`? (Or is it "text"?). That way it deals with the colors and hover states etc for you? Maybe you're getting something extra here, but it doesn't look like that to me 🤔

Reply:
I see you already add those variants in the implementation. Are the colors not right? There's also a `color` prop on mui buttons that you can use.